src/applecider/datasets/photo_dataset.py

```diff
@@ -115,29 +115,38 @@ def __len__(self):
         else:
             return len(self.filenames)
 
-
-def collate(batch):
-    '''custom collate function for photo events dataset'''
-    seqs = []
-    labels = []
-    for i in batch:
-        seqs += [i["data"]["photometry"]]
-        labels += [i["data"]["label"]]
-
-    lens = [s.size(0) for s in seqs]
-    pad  = pad_sequence(seqs, batch_first=True)            # (B, L, 7)
-    mask = torch.stack([torch.cat([torch.zeros(l), torch.ones(pad.size(1)-l)]) for l in lens]).bool()
-    # adjust padding mask to account for CLS at idx=0
-    pad_mask = torch.cat(
-            [torch.zeros(len(batch), 1, device=mask.device, dtype=torch.bool),
-             mask], dim=1
-        )
-    object_ids = [b["object_id"] for b in batch]
-    return {"data": {"photometry": pad,
-                     "label": torch.tensor(labels),
-                     "pad_mask": pad_mask,
-                     "mean": torch.tensor(batch[0]["data"]["mean"]),
-                     "std": torch.tensor(batch[0]["data"]["std"]),
-                     },
-            "object_id": object_ids,
```

Comment:
why don't we need object_id anymore? I recall that that was added for compliance with inference, has hyrax changed for this?